lib/iris/common/__init__.py

```diff
@@ -8,7 +8,121 @@
 
 """
 
+from typing import Mapping
+
 from .lenient import *
 from .metadata import *
 from .mixin import *
 from .resolve import *
+
+
+def filter_cf(
+    instances,
+    item=None,
+    standard_name=None,
+    long_name=None,
+    var_name=None,
+    attributes=None,
+):
+    """
+    Filter a list of :class:`iris.common.CFVariableMixin` subclasses to fit
+    the given criteria.
+
+    Kwargs:
+
+    * item
+        Either
+
+        (a) a :attr:`standard_name`, :attr:`long_name`, or
+        :attr:`var_name`. Defaults to value of `default`
+        (which itself defaults to `unknown`) as defined in
+        :class:`iris.common.CFVariableMixin`.
```

Comment:
@trexfeathers Save real-estate in the rendered docs by using,
```
`~iris.common.CFVariableMixin`
```
instead. This applies throughout 👍

Reply:
6675cd8a52c69f5b6f5cea55d93a86392c6ffd3d